src/nrnoc/treeset.cpp

```diff
@@ -499,8 +496,12 @@ void nrn_lhs(neuron::model_sorted_token const& sorted_token, NrnThread& nt) {
     }
 
     if (use_sparse13) {
-        // Zero the sparse13 matrix
-        spClear(_nt->_sp13mat);
+        Eigen::SparseMatrix<double, Eigen::RowMajor>& m_ = *_nt->_sparseMat;
+        for (int k = 0; k < m_.outerSize(); ++k) {
+            for (Eigen::SparseMatrix<double, Eigen::RowMajor>::InnerIterator it(m_, k); it; ++it) {
+                it.valueRef() = 0.;
+            }
+        }
```

Comment:
An other funny joke of `sparse13` the function `spClear` just reset all pre-declared number to 0 instead of removing them (it should be like that because this is a sparse matrix...).